modules/apps/portal-language/portal-language-override-service/src/main/java/com/liferay/portal/language/override/internal/PLOEntryIdentifiableOSGiService.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.language.override.internal;
+
+import com.liferay.portal.kernel.module.framework.service.IdentifiableOSGiService;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Joao Victor Alves
+ */
+@Component(service = IdentifiableOSGiService.class)
+public class PLOEntryIdentifiableOSGiService
```

Comment:
@MrJohn1911 This one shares the same issue as mentioned in https://github.com/liferay-core-infra/liferay-portal/pull/4855#pullrequestreview-1506703417

Reply:
see this [comment](https://github.com/liferay-core-infra/liferay-portal/pull/4855#discussion_r1248233160). Thanks.